src/string/memrchr.rs

```diff
@@ -0,0 +1,22 @@
+use c_types::{c_void, c_int, size_t};
+use core::ptr;
+
+#[linkage = "weak"]
```

Comment:
Is this working now? IIRC it was broken for a while.

Reply:
well, the symbols get marked as `W` in `nm librusl.a`, so I'd say yes it works 🤷‍♂️